conf/squid/rados/4-node-ec-cluster-1-client.yaml

```diff
@@ -0,0 +1,53 @@
+# Test Suite to test EC 8+6 MSR pool on 4 nodes
+# Deployment for all the ceph daemons , with 3 mon, 3 mgr, 24 OSD, 3 MDS, 3 RGW daemons
+globals:
+  - ceph-cluster:
+      name: ceph
+      node1:
+          role:
+            - _admin
+            - installer
+            - osd
+            - mon
+            - mgr
+            - rgw
+          no-of-volumes: 6
+          disk-size: 50
```

Comment:
Is it necessary to have such large disks?

Reply:
Since we only have 4 nodes, I felt we should have enough storage space.

Updated the capacity to 25G as other configs from RADOS. will update this if needed in future.